sys/net/gnrc/pktbuf_static/gnrc_pktbuf_static.c

```diff
@@ -380,11 +380,11 @@ static gnrc_pktsnip_t *_create_snip(gnrc_pktsnip_t *next, const void *data, size
             _pktbuf_free(pkt, sizeof(gnrc_pktsnip_t));
             return NULL;
         }
-    }
-    _set_pktsnip(pkt, next, _data, size, type);
-    if (data != NULL) {
         memcpy(_data, data, size);
     }
+
```

Comment:
Nitpick: Due to the added new lines this function call now unusually sticks out.

Reply:
sorry, fixed that.